addons/account/models/company.py

```diff
@@ -839,16 +840,14 @@ def build_move_info(move):
 
     def compute_fiscalyear_dates(self, current_date):
         """
-        The role of this method is to provide a fallback when account_accounting is not installed.
-        As the fiscal year is irrelevant when account_accounting is not installed, this method returns the calendar year.
-        :param current_date: A datetime.date/datetime.datetime object.
+        Returns the dates of the fiscal year containing the provided date for this company.
         :return: A dictionary containing:
             * date_from
             * date_to
         """
-
-        return {'date_from': datetime(year=current_date.year, month=1, day=1).date(),
-                'date_to': datetime(year=current_date.year, month=12, day=31).date()}
+        self.ensure_one()
+        date_from, date_to = date_utils.get_fiscal_year(current_date, day=self.fiscalyear_last_day, month=self.fiscalyear_last_month)
```

Comment:
@oco-odoo I think this should be:

```suggestion
        date_from, date_to = date_utils.get_fiscal_year(current_date, day=int(self.fiscalyear_last_day), month=int(self.fiscalyear_last_month))
```

If not, you get this error:

```
    fy_date_from = self.companies[0].compute_fiscalyear_dates(date_from_date)[
  File "/opt/odoo/auto/addons/account/models/company.py", line 849, in compute_fiscalyear_dates
    date_from, date_to = date_utils.get_fiscal_year(current_date, day=self.fiscalyear_last_day, month=self.fiscalyear_last_month)
  File "/opt/odoo/custom/src/odoo/odoo/tools/date_utils.py", line 72, in get_fiscal_year
    day = fix_day(date.year, month, day)
  File "/opt/odoo/custom/src/odoo/odoo/tools/date_utils.py", line 67, in fix_day
    max_day = calendar.monthrange(year, month)[1]
  File "/usr/local/lib/python3.10/calendar.py", line 125, in monthrange
    if not 1 <= month <= 12:
TypeError: '<=' not supported between instances of 'int' and 'str'
```

Reply:
@oco-odoo @smetl this is urgent, as it's breaking all the pipelines. Do we fill the corresponding PR?

Reply:
@pedrobaeza Indeed, sorry :/ Fixing now.